modules/client-common/test/unit/context.test.ts

```diff
@@ -25,7 +28,7 @@ describe("Context instances", () => {
     contextParams = {
       network: "mainnet",
       signer: new Wallet(TEST_WALLET),
-      daoFactoryAddress: "0x1234",
+      DAOFactory: "0x1234",
```

Comment:
For the future (new SDK), having an Address type would be helpful that makes sure that addresses are 40 chars long and lower case. (Probably such a type is provided by ethers).
Below, I see that also the empty string is allowed.